cmd/restic/integration_test.go

```diff
@@ -281,11 +282,21 @@ func testSetupBackupData(t testing.TB, env *testEnvironment) string {
 }
 
 func TestBackup(t *testing.T) {
+	t.Logf("Running normal test backup")
+	testBackup(t, false)
+
+	if runtime.GOOS == "windows" && fs.HasSufficientPrivilegesForVSS() {
+		t.Logf("Running additional test backup using filesystem snapshots")
+		testBackup(t, true)
```

Comment:
Please split that into a separate test case e.g. `TestBackupFsSnapshot`. It's very helpful for debugging to be able to test a specific functionality only.

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/0baa48293144a8a970716b58afaf1472e357e630

I still can't run integration tests locally, even when using a different version of tar I get different problems related to symlinks.

Reply:
Windows 10 has something called "Developer Mode" which allows a normal user to create symlinks. Normally that is restricted to applications running as administrator for security reasons.

Reply:
I've been running the tests as administrator using the same binaries for tar in my PATH. Not sure wrong on my system.